src/python/pants/backend/python/rules/pex.py

```diff
@@ -46,11 +46,15 @@ def create_pex(
     python_setup: PythonSetup,
     subprocess_encoding_environment: SubprocessEncodingEnvironment,
     pex_build_environment: PexBuildEnvironment,
-    platform: Platform
+    platform: PlatformConstraint
 ) -> Pex:
   """Returns a PEX with the given requirements, optional entry point, and optional
   interpreter constraints."""
 
+  # ignore none constraint if some rule pushes a constraint down as none.
```

Comment:
This is related to the `NB` below, I think? Because otherwise "unconstrained" would be ok. Should move it nearer the comment probably.